controllers/kmsissuer_controller.go

```diff
@@ -40,8 +40,6 @@ import (
 const (
 	// DefaultCertDuration is the default duration the CA certificate is valid for.
 	DefaultCertDuration = time.Hour * 24 * 365 * 3 // 3 years
-	// DefaultCertRenewalRatio is the default period of time before the CA cetificate is renewed.
-	DefaultCertRenewalRatio = 2 / 3
```

Comment:
I think it looks cleaner to have internal (unexported) constant instead "magic number" in the code.